libs/designsystem/accordion/src/accordion-item.component.spec.ts

```diff
@@ -72,4 +80,12 @@ describe('AccordionItemComponent', () => {
       });
     });
   });
+  it('should emit the toggle event when collapsed', () => {
+    spyOn(spectator.component.toggle, 'emit');
+    spectator.component.isDisabled = false;
+    spectator.component.isExpanded = true;
+    spectator.component._onToggleExpanded(new KeyboardEvent('keydown'));
+    expect(spectator.component.toggle.emit).toHaveBeenCalledTimes(1);
+    expect(spectator.component.toggle.emit).toHaveBeenCalledWith(false);
```

Comment:
Consider simplifying:
```suggestion
    expect(spectator.component.toggle.emit).toHaveBeenCalledOnceWith(false);
```